templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete car</h1>
+
+  <p>Are you sure you want to delete the car: {{ car }}</p>
+
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete" class="btn btn-danger">
```

Comment:
It would also be great if you added a cancelation button for every form. :)

Reply:
yeah, it's a great idea)